statsmodels/genmod/tests/test_glm.py

```diff
@@ -282,6 +284,28 @@ def test_compare_OLS(self):
 #        self.res2.resids = np.array(self.res2.resid)[:,None]*np.ones((1,5))
 #        self.res2.null_deviance = 185008826 # taken from R. Rpy bug?
 
+
+class TestGlmGaussianGradient(TestGlmGaussian):
+    @classmethod
+    def setup_class(cls):
+        '''
+        Test Gaussian family with canonical identity link
+        '''
+        # Test Precisions
+        cls.decimal_resids = DECIMAL_3
+        cls.decimal_params = DECIMAL_2
+        cls.decimal_bic = DECIMAL_0
+        cls.decimal_bse = DECIMAL_2
+
+        from statsmodels.datasets.longley import load
```

Comment:
I don't want to break the pattern for the `test_glm` module with loading the examples in the test classes instead of as module globals
glm and discrete cover many different cases and I never checked whether loading the datasets and results only once would be worth the extra memory usage.